toml/_parser.ts

```diff
@@ -894,7 +894,7 @@ export function ParserFactory<T>(parser: ParserComponent<T>) {
           if (count > line.length) {
             count -= line.length + 1;
           } else {
-            return count;
+            break;
```

Comment:
I use `break` here because `subStr.split("\n")` always returns a non-empty array. Therefore, the `for` loop is always called, which makes the line 900 unreachable.